src/Model/ActivityBasedTimeoutPolicy.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+* Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+* 
+* ActivityBasedTimeoutPolicy File
+* PHP version 7
+*
+* @category  Library
+* @package   Microsoft.Graph
+* @copyright © Microsoft Corporation. All rights reserved.
+* @license   https://opensource.org/licenses/MIT MIT License
+* @version   GIT: 1.4.0
+* @link      https://graph.microsoft.io/
```

Comment:
NIT: @darrelmiller, this seems to be a valid URL with proper redirect. But is there a more canonical URL that we want to promote?

Reply:
yep, this is now graph.microsoft.com. Add issue to track https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/issues/253